### PR TITLE
Add docker configuration

### PR DESCRIPTION
**Pull request by acoburn**
This adds a configuration for creating a docker container for Trellis and pushing that container to dockerhub.

### CODE REVIEW

**Comment by acoburn** [comment]
@gregjan would you be able to look at this Docker configuration? I cribbed from your addition to `trellis-ext-db`.

File: platform/linux/build.gradle [hunk 1]
```diff
@@ -61,6 +62,22 @@ buildDeb {
     preUninstall file('src/deb/preUninstall.sh')
 }
 
+docker {
+    name "trellisldp/trellis:${project.version}"
+    tags 'latest'
+    dockerfile file('src/docker/Dockerfile')
+    files tasks.distTar.outputs, './src/dist/etc/config.yml','./src/docker/command.sh'
```
Comment (gregjan) [inline platform/linux/build.gradle]: I think this line must be revised. It refers to the outputs of the "distTar" task, which I am pretty sure does not exist in stock trellis, at least I don't see it here.. The error I get upon Docker startup is that it cannot find "trellis" on the path.. "trellis" would have been packaged at /opt/trellis/bin/trellis if things had worked correctly.

**Comment by acoburn** [comment]
@gregjan Thanks! Yes, adding a console logger makes a lot of sense. I'd welcome any suggestions to improve on this, either via PR or just a pointer to working code/configuration.

File: platform/linux/build.gradle (hunk 1 above)
Comment (acoburn) [reply platform/linux/build.gradle]: Ok, the `distTar` task _should_ be present here, but I'll need to look at this more carefully.

**Pull request merged**